### DISCUSSION TITLE
Right understanding? Room and zones

### DISCUSSION THREAD

**Discussion by codingstube** [Hypfer/Valetudo#777] [Q&A / Support]
Good morning,

vendor: viomi

Do I unterstand it right, that I cannot select a room to clean. I can select one or more triangles on the map, but there is no start button. The "play" button in the map is for editing, isnt it? Feature is not implemented for viomi.

So I have to create zones for my rooms? In the xiaomi app there rooms and zones.

Thanks.

**Comment by Hypfer** [comment]
> The "play" button in the map is for editing, isnt it?

Uh.. no? Why do you think that? 
Also, why don't you press it and see for yourself? That would instantly answer this question without requiring additional resources and waiting for an answer.


Anyways, segment cleanup for viomi will become available with Valetudo 2021.04.0

**Comment by codingstube** [comment]
Hi Sören,

thanks for the answer. Sure I have clicked the play button. But a message popups that something like viomimapsegmentation is not implemented. So I thought that the play button will edit the map in the future.

So right now the play button has no function with viomi_v8 and I cannot clean rooms I have added in the xiaomi app. 
Zones I have added via valetudo works.

Maybe an other question. If the robot is connected to xiaomi cloud the robot shuts down after a while and it is not docked. If valetudo is running it stays "online". So there is no standby right now, isn't it?

**Reply by jloewe** [reply]
The ViomiMapSegmentationCapability was implemented a few days ago: https://github.com/Hypfer/Valetudo/pull/762

**Reply by codingstube** [reply]
Thanks. Works great. For me it is a big improvment.
